primitives/src/coin.rs

```diff
@@ -64,6 +61,14 @@ impl Coin {
         *self = self.safe_sub(rhs)?;
         Ok(())
     }
+
+    pub fn saturating_add(self, rhs: Coin) -> Coin {
+        Coin(self.0.saturating_add(rhs.0))
+    }
+
+    pub fn saturating_sub(self, rhs: Coin) -> Coin {
```

Comment:
I know it's self explanatory, but could you add a small `rustdoc`?

Reply:
Done.